ckanext/scheming/tests/mock_pylons_request.py

```diff
@@ -1,3 +1,4 @@
+from builtins import object
```

Comment:
what's this for?

Reply:
It's auto-fix provided by the `future` library - it provides exactly the same `object` for python2 as the one from python3. I don't think that we are relying on some low-level behavioral patterns of `object` that may be changed in python3, so it's likely to be unnecessary.